build/lib/common.sh

```diff
@@ -384,7 +383,6 @@ function retry_with_timeout() {
         # multiple the numeric part of the timeout by 1.5 and suffix with the last char which is the unit
         TIMEOUT=$((${TIMEOUT:0:-1} * 3/2))${TIMEOUT: -1}
         echo "Command failed. Attempt $n/$max with timeout ${TIMEOUT}:"
-        delay=$(( delay * 2 ))
```

Comment:
This was too aggressive for the codebuild checksum case where we expect lots of retries.  May bring this back in the future if we need to.